.circleci/create_circleci_config.py

```diff
@@ -156,6 +201,7 @@ def job_name(self):
         "pip install .[sklearn,torch,testing,sentencepiece,torch-speech,vision,timm]",
         "pip install git+https://github.com/huggingface/accelerate",
     ],
+    parallelism=8,
```

Comment:
This could be think of the maximal executors to use. The actual number is determined in job object's `to_dict`. For example, if the number of tests (at the levels of `tests/models/*` or `tests/pipelines/*`) are too  few, we will use fewer executors.